server/package.json

```diff
@@ -60,7 +60,7 @@
     "class-validator": "^0.14.0",
     "cookie-parser": "^1.4.6",
     "exiftool-vendored": "^28.3.1",
-    "fast-glob": "^3.3.2",
+    "fast-glob": "github:etnoy/fast-glob#built",
```

Comment:
If we want to patch this, lets do it in the immich-app repo. If this is the way we want to go we can create that fork.

Reply:
I'd be happy to do that. I've waited for weeks to get any response on my PR so I think it's time to fork